tests/fixtures/marc/marc_record_attribute_and_subfield_variations.xml

```diff
@@ -171,6 +183,124 @@
             <subfield code="r">r</subfield>
             <subfield code="t">t</subfield>
         </datafield>
+        <datafield tag="520">
+            <subfield></subfield>
+            <subfield code=""></subfield>
+            <subfield code="a">a</subfield>
+        </datafield>
+        <datafield tag="600">
+            <subfield></subfield>
+            <subfield code=""></subfield>
```

Comment:
Yes, that fixture is supposed to account for all variations for attributes and subfields. Because there could conceivably be subfields with no `code` attribute or no value, we just want to make sure the transform doesn't trip up on that

Reply:
@hakbailey could explain it better, I'm working off her templates for other sources